### ISSUE TITLE
Colors/Icons unclear when using Keybase Chat

### ISSUE THREAD

**Issue by CameronBanga** [keybase/keybase-issues#2817]
Downloaded the macOS version, playing around with the GUI a bit more, but I'm kinda confused at this point as to what username text color, and icons near photos mean. I see a lot of green and blue, and this seems to try to insinuate some sort of status, but I'm not sure what.

When I go to my user page to see who I follow/follows me, I see a bunch of photos, names, and dots/locks:

![screen shot 2017-02-08 at 6 24 28 pm](https://cloud.githubusercontent.com/assets/611354/22763706/02a4f182-ee2c-11e6-8d7b-14721265b9f5.png)

I'm presuming the lock and green text mean more secure/good? But I'm not certain. Likewise, my friend MangoSquash has blue text on this page for his handle, but when I've chatted him, his handle is green. I messaged another friend on the service, and his name is blue in chat. What should/do these colors indicate? I think them being different between the follower page and chat may be a bug, but I'm not sure.

Also not sure what the lock icon means. Mangosquash seems to have confirmed his identity on a couple services, at least as many as other people I follow who have a lock? Can't figure out what this means.

**Comment by cjb** [comment]
I think blue means you're following them, and Green means you're following them and they're following you. Does that clear it up?

**Comment by CameronBanga** [comment]
@cjb That makes sense. Mangosquash doesn't follow anyone, so he doesn't follow me.

But when I chat with him, or view his profile page, his name/background are green. When I view other people who do not follow me, their name/background seems to be blue. But I also found one or two users who both follow me and I follow, who and their name/backgrounds are blue. 

So not sure?

**Comment by cjb** [comment]
Thanks, does sound like a bug. We'll check it out.

**Comment by Mosesofmason** [comment]
Same issue. 

And what's the green dot and green circle mean?

**Comment by kognise** [comment]
And how about the lock?

**Comment by Dmium** [comment]
Sorry to bump but does anyone know what the orange means? I thought it was neither following but I have someone who follows me and I follow with an orange name

**Comment by riking** [comment]
Yes, I also have a mutual follower with a red/orange name. Shouldn't this be documented somewhere?

**Comment by heronhaye** [comment]
It means you followed them but one of the proofs you checked when you followed them is currently broken.

**Comment by riking** [comment]
Hmm, the proof break wasn't visible in the app yesterday, and I had no option to "Accept" + ignore broken proofs, but their name is green now. I suppose that means the proof got fixed.
